### PR TITLE
Update one_four_twentyfour.coffee

### PR DESCRIPTION
**Pull request by normandb85630**

### CODE REVIEW

**Comment by normandb85630** [comment]
BN

**Comment by normandb85630** [comment]
commit

File: scripts/one_four_twentyfour.coffee [hunk 1]
```diff
@@ -21,10 +21,13 @@
 
 
 sayings = [
-  'I need a new pair of shoes',
+
```
Comment (beardofedu) [inline scripts/one_four_twentyfour.coffee:24]:
```suggestion
'I need a new pair of shoes, with one Red Shoe',
```

Comment (beardofedu) [reply scripts/one_four_twentyfour.coffee:24]: Change this pls ☝️

**Review by beardofedu — CHANGES_REQUESTED** [review]
Um, fix the small issue and we are good to :ship:

🥂 🥂 🥂 🥂 🥂 🥂 🥂 🥂 🥂

**Comment by normandb85630** [comment]
fixed

**Pull request closed without merging**